src/bluesearch/entrypoint/database/schemas.py

```diff
@@ -0,0 +1,73 @@
+"""Module for defining SQL schemas."""
+
+# Blue Brain Search is a text mining toolbox focused on scientific use cases.
+#
+# Copyright (C) 2020  Blue Brain Project, EPFL.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from sqlalchemy import (
+    Boolean,
+    Column,
+    Date,
+    ForeignKey,
+    Integer,
+    MetaData,
+    String,
+    Table,
+    Text,
+    UniqueConstraint,
+)
+
+
+def schema_articles(metadata: MetaData) -> None:
+    """Add to the given 'metadata' the schema of the table 'articles'."""
+    Table(
+        "articles",
+        metadata,
+        Column("article_id", String(32), primary_key=True),
```

Comment:
Why 32?

Reply:
This is the length of the MD5 hash produced by `generate_uid(...)`.